theano/gpuarray/elemwise.py

```diff
@@ -41,6 +41,27 @@ def get_scal(dt):
     return scalar.get_scalar_type(dt)
 
 
+def max_inputs_to_GpuElemwise(node_or_outputs):
+    if isinstance(node_or_outputs, gof.Apply):
+        outputs = node_or_outputs.outputs
+    else:
+        outputs = node_or_outputs
+    ptr_size = 8
+    int_size = 4
```

Comment:
I did it for extra safety, but I think we start to limit too much fusion. This + the collapsing raise how much fusion we can do.

Not sure how high a priority this is...